NedyUdombat/ReachHigh
# Setup Infrastructure

#### What does this PR do?
Setup the Application infrastructure

#### Description of Task to be completed?
- [x] Install & Setup eslint
- [x] Install & Setup prettier
- [x] Install & Setup Babel
- [x] Setup Pull request template
- [x] Setup editor configuration
- [x] Setup Hound
- [x] Setup CodeClimate
- [x] Setup Coveralls

#### How should this be manually tested?
- Clone the repo,
- Check out to `ch-setup-infrastructure-169169055` branch
- Run `npm install` to install all dependencies,
- You should see all configurations

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?(if applicable)
[Pivotal tracker ID](https://www.pivotaltracker.com/story/show/169169055)

#### Screenshots
